guiEditor/src/diagram/guiNode.ts

```diff
@@ -102,19 +110,23 @@ export class GUINode {
     public constructor(globalState: GlobalState, public guiControl: Control) {
         this._globalState = globalState;
         this._ownerCanvas = this._globalState.workbench;
-        
+        this.x = guiControl.leftInPixels;
+        this.y = guiControl.topInPixels;
         guiControl.onPointerUpObservable.add(evt => {
             this.clicked = false;
             console.log("up");

```

Comment:
(not part of this PR, but...) is this console (and the others in this file)  needed?

Reply:
For now I'd like to keep it in since I still do a lot of debugging around that. Is that considered proper?

Reply:
As long as it goes out for production, i am totally fine with it. Maybe just add a TODO to follow up

Reply:
Totally!

Reply:
I'll make and issue for that.